### DISCUSSION TITLE
求无主之地2

### DISCUSSION THREAD

**Discussion by Vlary** [Gamuxorg/bbs#242] [投递和请求游戏]
游戏名称：无主之地2
游戏名称(英文)：Borderlands 2 
下载地址 :（如果有的话）：

**Comment by shouhuanxiaoji** [comment]
处理中

**Comment by shouhuanxiaoji** [comment]
https://www.linuxgame.cn/%e6%97%a0%e4%b8%bb%e4%b9%8b%e5%9c%b02
已处理
